tests/testthat/test-repwithna.R

```diff
@@ -0,0 +1,33 @@
+# helper data for testing
+df <- data.frame("name" = c("  ", ",.;", "?mark:"), "Age" = c(21,15,12))
+df_output_1 <- data.frame("name" = c(NA, ",.;", "?mark:"), "Age" = c(21, 15, 12), stringsAsFactors=FALSE)
+df_output_2 <- data.frame("name" = c(NA, NA, "?mark:"), "Age" = c(21, 15, 12), stringsAsFactors=FALSE)
+df_output_3 <- data.frame("name" = c(NA, ",.;", NA), "Age" = c(21, 15, 12), stringsAsFactors=FALSE)
+
```

Comment:
Might want to add a test to make sure function won't remove things its not supposed too by mistake (not that it does now, just in case), such that a string "Hello, World!" does't have its "," | " " | "!" removed by mistake

Reply:
Thanks Tani! I'll work on that. but maybe the second output can be a proof? as ",.;" is replaced but "?mark:" stays the same.